src/components/handbook-navigation/content.js

```diff
@@ -6,7 +6,7 @@ export const content = [
   { id: 4, link: "/community/handbook/repository-overview", text: "Repository Overview" },
   { id: 5, link: "/community/handbook/projects", text: "Projects" },
   { id: 6, link: "/community/handbook/mentorship-programs", text: "Mentorship Programs" },
-  { id: 7, link: "/community/handbook/learn-layer5", text: "Learn Layer5" },
+  { id: 7, link: "/community/handbook/learn-layer5", text: "Learning" },
```

Comment:
Link need to be changed here.

Reply:
In contributor-journey

Reply:
Thank you. This approach to internal anchoring is untenable. It should be dynamic, not manual.

Reply:
```suggestion
  { id: 7, link: "/community/handbook/contributor-journey", text: "Learning" },
```

Reply:
> Thank you. This approach to internal anchoring is untenable. It should be dynamic, not manual.

@YashKamboj something that you might look into?